contracts/Uni.sol

```diff
@@ -110,6 +110,22 @@ contract Uni {
         _moveDelegates(address(0), delegates[dst], amount);
     }
 
+    /**
+     * @notice Burns tokens
+     * @param rawAmount The number of tokens to be minted
```

Comment:
Minor - comment wording should indicate 'burnt', not 'minted'

Reply:
Yes of course, thanks for catching it!